server/controllers/users/userController.ts

```diff
@@ -0,0 +1,78 @@
+import { Response } from 'express';
+import Interviewee from '../../database/Models/Interviewee';
+import { RequestType } from '../../utils';
+import validateQuery from '../../utils/validation/queryValidation';
+
+const getAllReviews = async (req: RequestType, res: Response) => {
+  const { page = '1', saved = undefined } = req.query;
+
+  await validateQuery(req.query);
+  // Convert the incoming saved string into a boolean
+  const savedBoolean : any = saved ? JSON.parse(saved.toString())
+    : saved;
+
```

Comment:
You just Boolean the saved whenever you want to use it.

Reply:
If I booleaned the 'false' it will return true, and if i booleaned the false it will return false

Reply:
Your right they both none empty string, you can get it back from the validation as a boolean.